programs/autocrat_v0/src/lib.rs

```diff
@@ -151,147 +152,84 @@ pub mod autocrat_v0 {
         description_url: String,
         instruction: ProposalInstruction,
     ) -> Result<()> {
-        let openbook_pass_market = ctx.accounts.openbook_pass_market.load()?;
-        let openbook_fail_market = ctx.accounts.openbook_fail_market.load()?;
+        let pass_market = ctx.accounts.openbook_pass_market.load()?;
+        let fail_market = ctx.accounts.openbook_fail_market.load()?;
+        let base_vault = &ctx.accounts.base_vault;
+        let quote_vault = &ctx.accounts.quote_vault;
         let dao = &mut ctx.accounts.dao;
+        let clock = Clock::get()?;
 
         require!(
-            openbook_pass_market.base_mint
-                == ctx.accounts.base_vault.conditional_on_finalize_token_mint,
-            AutocratError::InvalidMarket
-        );
-        require!(
-            openbook_pass_market.quote_mint
-                == ctx.accounts.quote_vault.conditional_on_finalize_token_mint,
-            AutocratError::InvalidMarket
-        );
-
-        let current_time = Clock::get().unwrap().unix_timestamp as i64;
-
-        // The market expires a minimum of 7 days after the end of a 3 day proposal.
-        // Make sure to do final TWAP crank after the proposal period has ended
-        // and before the market expires, or else! Allows for rent retrieval from openbook
-        require!(
-            openbook_pass_market.time_expiry > current_time + TEN_DAYS_IN_SECONDS,
-            AutocratError::InvalidMarket
-        );
-        require!(
-            openbook_pass_market.seq_num == 0,
-            AutocratError::InvalidMarket
-        );
-        require!(
-            openbook_pass_market.taker_fee == dao.market_taker_fee,
-            AutocratError::InvalidMarket
-        );
-        require!(
-            openbook_pass_market.maker_fee == 0,
-            AutocratError::InvalidMarket
-        );
-        require!(
-            openbook_pass_market.base_lot_size == dao.base_lot_size,
-            AutocratError::InvalidMarket
-        );
-        require!(
-            openbook_pass_market.quote_lot_size == 100, // you can quote META in increments of a hundredth of a penny
-            AutocratError::InvalidMarket
-        );
-        require!(
-            openbook_pass_market.collect_fee_admin == dao.treasury,
-            AutocratError::InvalidMarket
-        );
-
-        require!(
-            openbook_fail_market.base_mint
-                == ctx.accounts.base_vault.conditional_on_revert_token_mint,
-            AutocratError::InvalidMarket
-        );
-        require!(
-            openbook_fail_market.quote_mint
-                == ctx.accounts.quote_vault.conditional_on_revert_token_mint,
-            AutocratError::InvalidMarket
-        );
-        require!(
-            openbook_fail_market.time_expiry > current_time + TEN_DAYS_IN_SECONDS,
-            AutocratError::InvalidMarket
-        );
-        require!(
-            openbook_fail_market.seq_num == 0,
```

Comment:
This check is missing in the new tests

Reply:
intentional, [here's my justification](https://github.com/metaDAOproject/futarchy/pull/104/commits/1cfb3c498084d771d7211492f6096e5b3a33e701)

Reply:
got it